scripts/game_structure/ui_elements.py

```diff
@@ -536,6 +542,21 @@ def _get_scroll_bar_width(self) -> int:
     def _get_scroll_bar_height(self) -> int:
         return ui_scale_value(18) + 2
 
+    def are_contents_hovered(self) -> bool:
+        """
+        Are any of the elements in the container hovered? Used for handling mousewheel events.
+
+        :return: True if one of the elements is hovered, False otherwise.
+        """
+        any_hovered = False
+        for element in self:
+            if any(sub_element.hovered for sub_element in element.get_focus_set()):
+                any_hovered = True
+            elif isinstance(element, IContainerLikeInterface):
+                any_hovered = element.are_contents_hovered()
+            if any_hovered:
+                break
+        return any_hovered
```

Comment:
```suggestion
        for element in self:
            if any(sub_element.hovered for sub_element in element.get_focus_set()):
                return True
            elif isinstance(element, IContainerLikeInterface) and element.are_contents_hovered():
                return True
        return False
```
I think my indents are correct (and that this all checks out?)